code/game/gamemodes/vampire/vampire.dm

```diff
@@ -5,6 +5,21 @@
 	var/list/datum/mind/vampire_enthralled = list() //those controlled by a vampire
 	var/list/vampire_thralls = list() //vammpires controlling somebody
 
+
+/mob/living/carbon/human/Stat()
+	. = ..()
+	var/datum/antagonist/vampire/vamp = mind.has_antag_datum(/datum/antagonist/vampire)
```

Comment:
Wouldn't you also put this in an `if(is_vampire(src))` as the proc below?

Reply:
No, that's not needed. The `vamp` datum is needed here in order to display information, and if the `vamp` datum doesn't exist, it won't be displayed anyways.

Reply:
Given how common a task it is that antagonists add information to the "Status" panel, I think it would be wise to add a handler to the `/datum/antagonist` datum that took care of presenting this information. Then, in `human.dm`, you can loop through all antag datums on the mob, and call their `Stat` handler. Adding another implementation of `human/Stat` should be avoided where possible.